### DISCUSSION TITLE
problem whith shorten button

### DISCUSSION THREAD

**Discussion by saidNaa** [YOURLS/YOURLS#3601] [General]
Hello, I installed and configured yourls on my server. Everything works fine when I use the bookmark to shorten the links but when I use the shorten button on the  interface  page (both as admin or as normal user)   the created short link (i.e. myshort.com/keyword) does not point to the desired long link (i.e. www.some_site.com) but instead points to the following link : https://myshort.com/admin/www.some_site.com,  and of course it goes nowhere. I'm literally freaking out and I'm asking for some help please. My configuration file is the following:

define( 'YOURLS_DB_USER', 'myself' );

define( 'YOURLS_DB_PASS', 'mypassword' );

define( 'YOURLS_DB_NAME', 'myshort' );

define( 'YOURLS_DB_HOST', 'localhost' );

define( 'YOURLS_DB_PREFIX', 'myshort_' );

define( 'YOURLS_SITE', 'https://myshort.com' );

define( 'YOURLS_LANG', '' );

define( 'YOURLS_UNIQUE_URLS', false );

define( 'YOURLS_PRIVATE', true );

define( 'YOURLS_COOKIEKEY', 'y7dA7[UQ5Zp77K4-2wJTb$SB((McOD&sJRETv~w)' );

$yourls_user_passwords = [
        'xxx1' => 'phpass:xxxxxxxxx/* Password encrypted by YOURLS */ ,
         'xxx2' => 'phpass:xxxxxxxx /* Password encrypted by YOURLS */ ,
        // You can have one or more 'login'=>'password' lines
];

define( 'YOURLS_URL_CONVERT', 36 );

define( 'YOURLS_DEBUG', true );

$yourls_reserved_URL = [
        'porn', 'faggot', 'sex', 'nigger', 'fuck', 'cunt', 'dick',
];

Thank you very much

**Comment by dgw** [comment]
> points to the following link : https://myshort.com/admin/www.some_site.com

Don't omit the protocol.

**Reply by saidNaa** [reply]
What a fool i am. Thanks very much, works perfectly.
